lib/src/components/NxCollapsibleItems/NxCollapsibleItems.tsx

```diff
@@ -46,11 +46,12 @@ const NxCollapsibleItems: NxCollapsibleItemsFC = function NxCollapsibleItems(pro
         'nx-collapsible-items--empty': isEmpty
       }),
       treeViewId = useUniqueId('nx-collapsible-items', id),
+      treeViewChildrenId = useUniqueId('nx-collapsible-items__children'),
```

Comment:
BEM is not used in ids
```suggestion
      treeViewChildrenId = useUniqueId('nx-collapsible-items-children'),
```